modules/appeals_api/app/swagger/appeals_api/v2/swagger.json

```diff
@@ -3089,26 +3089,31 @@
             "required": true,
             "description": "Scoping of appeal type for associated issues",
             "schema": {
-              "type": "string"
+              "type": "string",
+              "enum": [
+                "higher_level_reviews",
+                "notice_of_disagreements",
+                "supplemental_claims"
+              ]
             }
           },
           {
             "name": "benefit_type",
             "in": "query",
-            "description": "Benefit Type for the appeal. Required if Decision Review is a Higher Level Review.",
-            "enum": [
-              "compensation",
-              "pensionSurvivorsBenefits",
-              "fiduciary",
-              "lifeInsurance",
-              "veteransHealthAdministration",
-              "veteranReadinessAndEmployment",
-              "loanGuaranty",
-              "education",
-              "nationalCemeteryAdministration"
-            ],
+            "description": "Required if decision review type is Higher Level Review or Supplemental Claim.",
```

Comment:
Let's not mention SC's here on prod documentation until we've done E2E testing in the lower envs & released the endpoint to prod.

Reply:
oh right!